testsuite/features/secondary/srv_distro_cobbler.feature

```diff
@@ -140,7 +140,8 @@ Feature: Cobbler and distribution autoinstallation
     And I wait until file "/srv/tftpboot/pxelinux.0" exists on server
 
   Scenario: Trigger the creation of a cobbler system record
-    When I trigger cobbler system record on the "sle_minion"
+    When I wait for "300" seconds
```

Comment:
No pleaseeee! This wait no!!! :D 
Try to use a step that use a timeout, checking it periodically, but avoid waiting idle for 300 seconds

Reply:
Another Net Timeout, in the test runs it didn't have anything to check on, I could not find any other solution
But once again it might not be needed in the main pipelines

Reply:
Following the discussion [on that topic](https://suse.slack.com/archives/C02D12TNYLS/p1665740743091379) I will remove this wait once everything else is ready along with the other extra explicit waits